straxen/plugins/high_energy_processing.py

```diff
@@ -0,0 +1,121 @@
+import strax
+import straxen
+import numpy as np
+
+export, __all__ = strax.exporter()
+
+# These are also needed in peaklets, since hitfinding is repeated
+HITFINDER_OPTIONS_he = tuple([
+    strax.Option(
+        'hit_min_amplitude_he',track=False,
+        default=tuple(np.ones(800, dtype=np.int16) * 15),
+        help='Minimum hit amplitude in ADC counts above baseline. '
+             'See straxen.hit_min_amplitude for options.'
+    )])
+
+
+@export
+@strax.takes_config(
+    *HITFINDER_OPTIONS_he)
+class PulseProcessingHe(straxen.PulseProcessing):
+    __version__ = '0.2.5'
+    rechunk_on_save = False
+    depends_on = 'raw_records_he'
+    provides = 'records_he'
+    data_kind = 'records_he'
+
+    def infer_dtype(self):
+        dtype = strax.record_dtype(strax.DEFAULT_RECORD_LENGTH)
+        return dtype
+
+    def setup(self):
+        self.hev_enabled=False
+        self.config['hit_min_amplitude'] = self.config['hit_min_amplitude_he']
+
+    def compute(self, raw_records_he, start, end):
+        result = super().compute(raw_records_he, start,end)
+        return result['records']
+
+    
+@export
+@strax.takes_config(
+
+    strax.Option(
+        'n_tpc_pmts_all',track=False,default=800
+        ),
+    *HITFINDER_OPTIONS_he
+)
+class PeakletsHe(straxen.Peaklets):
+    depends_on = 'records_he'
+    provides = 'peaklets_he'
+    data_kind = 'peaklets_he'
+    __version__ = '0.0.3'
+    parrallel=False
+    
+    def infer_dtype(self):
+        return strax.peak_dtype(
+                        n_channels=self.config['n_tpc_pmts_all'])
+
+    def setup(self):
+        self.config['hit_min_amplitude'] = self.config['hit_min_amplitude_he']  
+        self.to_pe = straxen.get_to_pe(self.run_id,
+                                       self.config['gain_model'],
+                                       n_tpc_pmts=self.config['n_tpc_pmts'])
+        buffer_pmts = np.zeros(500)
+        self.to_pe = np.concatenate((buffer_pmts,self.to_pe))
+        self.to_pe *= 20
```

Comment:
better add this to an option ?

Reply:
In principle this is something we could store somewhere once we've calibrated the per-channel gain difference. It's going to be roughly 20, but we don't know the spread.

Reply:
Also shouldn't we be dividing by 20 rather than multiplying?

Reply:
The signal is 20 times smaller. So 1 pe in a he channel is 20 real pe's

Reply:
Could you make this number an option? Kind of similar to the hitfinder threshold? There you can specify either an array or just a single number. https://github.com/AxFoundation/strax/blob/0f90d8c6dc520d5747e5108957ac7044a580274d/strax/processing/pulse_processing.py#L194

Reply:
this is already done